src/solidation/__main__.py

```diff
@@ -351,6 +352,56 @@ def to_markdown(self) -> str:
             pr_durations = [(i.merged_at - i.created_at).days for i in merged_prs]
             if len(pr_durations) > 1:
                 s += f"- PR duration quantiles (days): {quantiles(pr_durations)}\n"
+
+        if self.config.members:
+            s += "##### Issues & PRs assigned to each member:\n"
+            for user in sorted(self.config.members):
+                open_qty = sum(
+                    1
+                    for ip in self.open_ip
+                    if any(u.login == user for u in ip.assignees)
+                )
+                open_pr_qty = sum(
+                    1
+                    for pr in self.open_prs
+                    if any(u.login == user for u in pr.assignees)
+                )
+                open_issue_qty = open_qty - open_pr_qty
+                blocked_qty = sum(
+                    1
+                    for ip in self.open_ip
+                    if any(u.login == user for u in ip.assignees)
+                    and any(lbl.name == "blocked" for lbl in ip.labels)
+                )
+                new_assigned_qty = 0
+                new_closed_qty = 0
+                for ip in self.active_issues + self.active_prs:
+                    if any(u.login == user for u in ip.assignees):
+                        if ensure_aware(ip.created_at) >= since:
+                            new_assigned_qty += 1
+                        if (
+                            ip.closed_at is not None
+                            and ensure_aware(ip.closed_at) >= since
+                        ):
+                            new_closed_qty += 1
+                since_query = quote(since.isoformat(timespec="seconds"))
+                s += (
+                    f"- [{user}](https://github.com/{user}):"
+                    f" [{open_qty}](https://github.com/issues?"
```

Comment:
this number does not correspond (a bit lower) than the number on the target page. Any idea why?

Reply:
I was looking at https://github.com/con/solidation-con/blob/master/reports/2023-03-16.md#issues--prs-assigned-to-each-member and e.g. for you it shows
```
jwodder: 39 = 38 issues/1 PRs; 7 blocked +2/-2
```
but clicking on 38 issues leads to https://github.com/issues?q=assignee%3Ajwodder+is%3Aopen+type%3Aissue with 42 items. I found listing of jwodder/wheel-inspect . Not sure I spot any other but the point is the same -- indeed that page would list other repos as well, so expected to possibly have more of them which is ok with me (giving full list of repos even if possible would be too ugly)